sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/implementation/MessageIdsImpl.java

```diff
@@ -22,121 +22,148 @@
 import com.azure.storage.queue.implementation.models.MessageIdsDeleteResponse;
 import com.azure.storage.queue.implementation.models.MessageIdsUpdateResponse;
 import com.azure.storage.queue.implementation.models.QueueMessage;
-import com.azure.storage.queue.models.QueueStorageException;
+import com.azure.storage.queue.implementation.models.StorageErrorException;
 import reactor.core.publisher.Mono;
 
-/**
- * An instance of this class provides access to all the operations defined in
- * MessageIds.
- */
+/** An instance of this class provides access to all the operations defined in MessageIds. */
 public final class MessageIdsImpl {
-    /**
-     * The proxy service used to perform REST calls.
-     */
-    private MessageIdsService service;
+    /** The proxy service used to perform REST calls. */
+    private final MessageIdsService service;
 
-    /**
-     * The service client containing this operation class.
-     */
-    private AzureQueueStorageImpl client;
+    /** The service client containing this operation class. */
+    private final AzureQueueStorageImpl client;
 
     /**
      * Initializes an instance of MessageIdsImpl.
      *
      * @param client the instance of the service client containing this operation class.
      */
-    public MessageIdsImpl(AzureQueueStorageImpl client) {
-        this.service = RestProxy.create(MessageIdsService.class, client.getHttpPipeline());
+    MessageIdsImpl(AzureQueueStorageImpl client) {
+        this.service =
+                RestProxy.create(MessageIdsService.class, client.getHttpPipeline(), client.getSerializerAdapter());
         this.client = client;
     }
 
     /**
-     * The interface defining all the services for AzureQueueStorageMessageIds
-     * to be used by the proxy service to perform REST calls.
+     * The interface defining all the services for AzureQueueStorageMessageIds to be used by the proxy service to
+     * perform REST calls.
      */
     @Host("{url}")
-    @ServiceInterface(name = "AzureQueueStorageMessageIds")
+    @ServiceInterface(name = "AzureQueueStorageMes")
     private interface MessageIdsService {
```

Comment:
You'll want to add the Swagger directive mentioned in this [comment](https://github.com/Azure/azure-sdk-for-java/issues/17368#issuecomment-774356427) to resolve the issue present in #12829.

Reply:
Done